grails-app/views/order/_actions.gsp

```diff
@@ -76,27 +76,27 @@
 				</div>
 			</g:if>
 			<g:isSuperuser>
-				<div class="action-menu-item">
-					<hr/>
-				</div>
-				<div class="action-menu-item">
-					<g:link controller="order" action="rollbackOrderStatus" id="${orderInstance?.id}"
-						disabled="${!(orderInstance?.isReceived()||orderInstance?.isCompleted())}"
```

Comment:
I did not saw anything in the ticket about changing the rollback and before this change, you could rollback received and completed putaways.